### PR TITLE
How to change the 'time' parameter of the deduplication handler

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Don't know how this needs to be written in XML & PHP, sorry.

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]
I just pushed an example for the XML and PHP format.

File: logging/monolog_email.rst [hunk 1]
```diff
@@ -28,6 +28,8 @@ it is broken down.
                     handler:      deduplicated
                 deduplicated:
                     type:    deduplication
+                    # uncomment this line for values other than 60 seconds:
```
Comment (javiereguiluz) [inline logging/monolog_email.rst]: This comment and the provided `time: 10` example looks a bit confusing to me. What if we explain this option more in detail and don't include the `time: 10` example commented?

```yaml
deduplicated:
    type:    deduplication
    # the time in seconds during which duplicate entries are discarded (default: 60)
    time: 10
    handler: swift
```

**Comment by ThomasLandauer** [comment]
@javiereguiluz Don't know what you mean exactly.

My motivation for committing this: Right now, the `time` setting is explained nowhere. I basically guessed how to write it ;-) The existence of the setting is mentioned a few paragraphs further down:

> The deduplicated handler simply keeps all the messages for a request and then passes them onto the nested handler in one go, but only if the records are unique over a given period of time (60 seconds by default).

So I figured I could squeeze it right into the first code block. Not ideal, but better than nothing :-)

However, we could certainly create a separate code block for this, maybe something like:
````
monolog:
    handlers:
        deduplicated:
            type:    deduplication
            handler: swift
            time: 10

... and position it right below the cited paragraph.

File: logging/monolog_email.rst (hunk 1 above)
Comment (HeahDude) [inline logging/monolog_email.rst]: I like @javiereguiluz suggestion.

**Review by HeahDude — APPROVED** [review]

**Pull request merged**